ansible/Dockerfile

```diff
@@ -1,14 +1,9 @@
-FROM alpine:3.8
+FROM python:2.7-alpine3.8
 
-ARG CI_HELPER_VERSION
-ARG PYTHON_VERSION
-ARG VERSION
+ARG VERSION="2.5.5"
 
 WORKDIR /root
 RUN echo "Install Ansible with deps" && \
-    apk --update --no-cache add bash ca-certificates curl git openssh-client python=${PYTHON_VERSION} py-pip ansible=${VERSION} && \
-    echo "Done install Ansible with deps" &&\
-    echo "Install CI Helper" && \
-    curl -sSL https://github.com/rande/gitlab-ci-helper/releases/download/${CI_HELPER_VERSION}/alpine-amd64-gitlab-ci-helper -o /usr/bin/ci-helper && \
-    chmod 755 /usr/bin/ci-helper && \
-    echo "Done install CI Helper"
+    apk add --update -q --no-cache bash build-base curl git libffi-dev make openssh-client rsync tzdata openssl-dev && \
```

Comment:
What are `build-base`, `libffi-dev`, `make` and `openssl-dev` for? Can they be removed after `pip install`?

Maybe also remove the Alpine index files at the end?

Reply:
Nop, we need this to be able to compile some python library using Cpython. As this image maybe use with other python script or python lib, we need these libs directly in the source image.

Reply:
OK for the packages, but can we still add `rm -rf /var/cache/apk/*` at the end?